internal/services/containers/kubernetes_cluster_node_pool_data_source.go

```diff
@@ -54,6 +54,11 @@ func dataSourceKubernetesClusterNodePool() *pluginsdk.Resource {
 				Computed: true,
 			},
 
+			"gpu_profile": {
```

Comment:
Can we rename this field to `gpu_driver` which is more descriptive and consistent with az cli.